models/actions/runner.go

```diff
@@ -52,14 +52,50 @@ type ActionRunner struct {
 	Deleted timeutil.TimeStamp `xorm:"deleted"`
 }
 
-func (r *ActionRunner) OwnType() string {
+// RunnerType defines the runner type
+type RunnerType int
+
+const (
+	// RunnerTypeGlobal defines a global runner
+	RunnerTypeGlobal RunnerType = iota
+
+	// RunnerTypeOrganization defines an organization runner
+	RunnerTypeOrganization
+
+	// RunnerTypeRepository defines a repository runner
+	RunnerTypeRepository
+)
+
+func (r *ActionRunner) Type() RunnerType {
+	if r.RepoID != 0 {
+		return RunnerTypeRepository
+	}
+	if r.OwnerID != 0 {
+		return RunnerTypeOrganization
+	}
+	return RunnerTypeGlobal
+}
+
+func (rt RunnerType) String() string {
+	switch rt {
+	case RunnerTypeGlobal:
+		return "Global"
+	case RunnerTypeOrganization:
+		return "Organization"
+	case RunnerTypeRepository:
+		return "Repository"
+	}
+	return fmt.Sprintf("Unknown Type %d", rt)
+}
+
+func (r *ActionRunner) BelongsTo() string {
 	if r.RepoID != 0 {
-		return fmt.Sprintf("Repo(%s)", r.Repo.FullName())
+		return r.Repo.FullName()
 	}
 	if r.OwnerID != 0 {
-		return fmt.Sprintf("Org(%s)", r.Owner.Name)
+		return r.Owner.Name
```

Comment:
`Name` or `FullName`?

Reply:
`FullName` is empty by default.
It should be `Name`.